pages/activity/[id].js

```diff
@@ -23,27 +25,34 @@ export default function SingleActivity({ editTag }) {
 
         <li>
           <TagList
+            tags={activity.tags}
             activityId={activity.id}
             tagId={activity.tags}
             editTag={editTag}
           />
         </li>
 
-        {activity.imageURL && <TagImg src={activity.imageURL} />}
-        <li>Date: {activity.date}</li>
+        {activity.imageURL && <Img src={activity.imageURL} />}
```

Comment:
As name for a styled component we prefer to not use the html element tag name if this name is an abbreviation. `Image` would therefore be a better name. :)